src/frontend/src/session/cursor_manager.rs

```diff
@@ -436,7 +457,12 @@ impl SubscriptionCursor {
             );
         }
         while cur < count {
+            let fetch_cursor_timer = Instant::now();
             let row = self.next_row(&handle_args, formats).await?;
+            self.cursor_metrics
+                .subscription_cursor_fetch_duration
```

Comment:
The metric here is not measuring the whole fetch duration. How about doing it in `fetch_cursor.rs`?

Reply:
We may fetch n rows at a time, which in `fetch_cursor.rs` statistics may lead to inconsistent metrics and less overhead for methods other than `next_row()`